packages/data-transport-layer/src/services/server/service.ts

```diff
@@ -128,6 +128,7 @@ export class L1TransportServer extends BaseService<L1TransportServerOptions> {
         this.logger.info('Served HTTP Request', {
           method: req.method,
           url: req.url,
+          body: json,
```

Comment:
Is this the full response data? Couldn't this be quite large?

Reply:
before mark's comment i went through all the routes and logged statuses from there, but logging body here accomplishes nearly the same thing (checking whether data is being fetched correctly) without the overhead. i could make this a debug log or remove it, thoughts?

Reply:
This could be quite large and may not be necessary, if the data was not fetched correctly then we should be sure that an error is thrown so that the error handler logger catches the error and logs the message

Reply:
The one useful thing about it is that we will have in our logs the data that is being fed to our services so it could be useful for that

Reply:
added this back in as a debug log!